src/extractor/guidance/intersection_generator.cpp

```diff
@@ -176,7 +176,7 @@ Intersection IntersectionGenerator::mergeSegregatedRoads(Intersection intersecti
         const auto &first_data = node_based_graph.GetEdgeData(intersection[first].turn.eid);
         const auto &second_data = node_based_graph.GetEdgeData(intersection[second].turn.eid);
 
-        return first_data.name_id != INVALID_NAME_ID && first_data.name_id == second_data.name_id &&
+        return first_data.name_id != INVALID_NAMEID && first_data.name_id == second_data.name_id &&
```

Comment:
@MoKob Are you sure this is the correct change?  Should we be testing for an INVALID_NAMEID (UINT_MAX), or the empty name (0)?